server/src/test/java/org/elasticsearch/action/get/GetRequestTests.java

```diff
@@ -40,7 +40,7 @@ public void testValidation() {
             final ActionRequestValidationException validate = request.validate();
 
             assertThat(validate, not(nullValue()));
-            assertThat(validate.validationErrors(), hasItems("type is missing", "id is missing"));
+            assertThat(validate.validationErrors(), hasItems("id is missing"));
```

Comment:
I don't think we need this test change.

Reply:
Yes, thats a left-over from the validation I removed and then added again. Will update.